YOLayout/YOLayout_IMP.h

```diff
@@ -308,4 +314,14 @@ typedef CGSize (^YOLayoutBlock)(id<YOLayout> layout, CGSize size);
  */
 + (YOLayout *)layoutWithLayoutBlock:(YOLayoutBlock)layoutBlock;
 
+#pragma mark Common Layouts
+
++ (YOLayout *)vertical:(id)view;
```

Comment:
Neat. Add some comments!

Let's make a mental note to watch how often these are useful in practice.